### ISSUE TITLE
Evaluate if BossDB is a good fit for Nova

### ISSUE THREAD

**Issue by burbas** [novaframework/nova#216]
Instead of trying to work on (yet) another project we could see if we can take advantage of BossDB instead (https://github.com/ErlyORM/boss_db).

**Comment by Maartz** [comment]
Hello

I was browsing Nova's codebase and didn't see any reference to DB management. Maybe I missed it.
Nevertheless, I was wondering what would be the pros/cons for adding BossDB.
I'm really eager to understand better what you aim for and which developer experience you want to have.

Thanks :D

**Comment by Taure** [comment]
We did have a version where we used [ErlDB](https://github.com/erldb/erldb ) but we did feel we didn't get a good structure and remove it. So today I would say that Nova is more of a View-Controller framework, where we haven't solved the modelling part. Phoenix has ecto.

So right now we will see if we develop that again or find a way to handling the modelling part better, like see if we can use stuff from bossdb or not.
